doc/en/reference.rst

```diff
@@ -1872,6 +1872,9 @@ All the command-line flags can be obtained by running ``pytest --help``::
                             log date format as used by the logging module.
```

Comment:
Please also add the new option to the `Configuration Options` in the same file. 👍